test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -114,7 +115,7 @@
     ('aten::__and__', datetime.date(2020, 6, 30)),
     ('aten::__or__', datetime.date(2020, 6, 30)),
     ('aten::__xor__', datetime.date(2020, 6, 30)),
-    ('aten::split', datetime.date(2020, 6, 30)),
+    ('aten::split', datetime.date(2020, 7, 20)),
```

Comment:
This one you're not changing the signature, so you can actually just remove the line :D